main.go

```diff
@@ -71,7 +77,10 @@ func main() {
 		SilenceUsage:  true,
 	}
 
+	cobra.OnInitialize(initConfig)
+
 	// Set up the cobra root command flags
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file")
```

Comment:
The comment here should be more descriptive. "Use this flag to override the default config file lookup" or something along those lines